changelogs/unreleased/gh-4726-execute-field-in-box.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/box
+
+* **[Breaking change]** Now the "execute" field is not available until BOX is
+  configured (gh-4726).
```

Comment:
Shouldn't we also add somewhere that `box.execute()` does not automatically configures `box`? It seems as a more important fact for a user.

Reply:
I've added a doc-request that `execute()` cannot be called until the box has been initialized. I think this should be enough.

Reply:
I meant the wording in the changelog entry, but anyway it is already too picky. Closing the discussion.